src/service/loader.js

```diff
@@ -0,0 +1,66 @@
+import log from "loglevel";
+import { Agent } from "https";
+import { readFileSync } from "fs";
+
+export default class setup {
+  constructor(dotenv) {
+    let logLevel = process.env.LOG_LEVEL || "DEBUG";
+    log.setLevel(log.levels[logLevel]);
+
+    const result = dotenv.config();
+    if (result.error) {
+      throw result.error;
+    }
+    console.log("Environment variables loaded.");
+
+    this.url = process.env.MESH_URL || "https://localhost:8700";
+    this.sharedKey = process.env.MESH_SHARED_KEY || "TestKey";
+    this.sandbox = process.env.MESH_SANDBOX || "true";
```

Comment:
true should be a bool not a string

Reply:
Environment variables are strings by default, hence the default is a string "true".